src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -586,13 +586,61 @@ await invokeTask.NoThrowAwaitable(); // may or may not throw due to cancellation
     [Fact]
     public void JsonRpcMethodAttribute_ConflictOverloadMethodsThrowsException()
     {
-        var invalidServer = new InvalidOverloadServer();
+        var invalidServer = new ConflictingOverloadServer();
 
         var streams = Nerdbank.FullDuplexStream.CreateStreams();
         var serverStream = streams.Item1;
         var clientStream = streams.Item2;
 
-        Assert.Throws<RemoteTargetException>(() => JsonRpc.Attach(this.serverStream, invalidServer));
+        Assert.Throws<ArgumentException>(() => JsonRpc.Attach(this.serverStream, invalidServer));
```

Comment:
Thanks for testing the failure case.
Can you also test that `new JsonRpc(...)` would also throw?